src/app/static/src/tests/components/runWorkflow/runWorkflowRun.test.ts

```diff
@@ -36,12 +37,11 @@ describe(`runWorkflowRun`, () => {
             .reply(200, {"data": workflowSummaryMetadata});
     })
 
-    const getWrapper = (propsData: any = {}) => {
+    const getWrapper = (propsData: Partial<RunWorkflowMetadata> = {}) => {
```

Comment:
tiny change but can we rename `propsData` here to `workflowMetadata` just to be super clear that this ins't the whole `propsData` object?